sale_subscription_financial_risk/data/sale_subscription_financial_risk_suspend.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" ?>
+<odoo>
+    <data noupdate="1">
+        <!-- scheduled action to batch check all subscription states for financial risk violation and suspend/re-activate as needed -->
+        <record id="check_service_suspensions" model="ir.cron">
+            <field name="name">Suspended Services Check</field>
+            <field name="model_id" ref="model_sale_subscription"/>
+            <field name="active" eval="True" />
+            <field name="user_id" ref="base.user_root" />
+            <field name="interval_number">1</field>
+            <field name="interval_type">days</field>
+            <field name="numbercall">-1</field>
+            <field name="priority">1</field>
+            <field name="doall" eval="False"/>
+            <field name="state">code</field>
+            <field name="code">
+records = model.search([])
+for record in records:
+    if record.stage_id.name in ['In Progress', 'Suspended']:
```

Comment:
Just a comment: We should never rely on Static Name, maybe based on  xml_id

Reply:
By `xml_id` I mean `self.env.ref('module_name.xml_stage_id') `so we can have dynamic Id in that case